src/os_unix.c

```diff
@@ -670,7 +670,7 @@ mch_delay(long msec, int flags)
 	}
 #  endif // HAVE_SELECT
 # endif // HAVE_NANOSLEEP
-#endif // HAVE_USLEEP
+#endif // HAVE_NANOSLEEP
```

Comment:
It should be:
```
   672 # endif // HAVE_USLEEP
   673 #endif // HAVE_NANOSLEEP
```

Reply:
Whoops. *blush* Just pushed 76eca116a to fix this.